apps/condo/domains/common/components/Import/BaseImportWrapper.tsx

```diff
@@ -0,0 +1,278 @@
+                        type='info'
+                        message={RequiredFieldsTitle}
+                        description={ImportRequiredFieldsMessage}
+                    />
+                    <LinkWithIcon
+                        title={ExampleLinkMessage}
+                        size='medium'
+                        PrefixIcon={Download}
+                        href={exampleTemplateLink}
+                        target='_blank'
+                    />
+                </Space>
+            </Modal>
+            <Modal
+                title={ProgressModalTitle}
+                open={activeModal === 'progress'}
+                onCancel={handleBreakImport}
+            >
+                <StyledFocusContainer>
+                    <Row gutter={[0, 32]} justify='center' align='middle'>
+                        <Col>
+                            <img alt='progress-image' src='/progressDino.png' />
+                        </Col>
+                        <Col>
+                            <Space size={16} direction='vertical'>
+                                <Typography.Title level={3}>{ProgressModalMessage}</Typography.Title>
+                                <Typography.Text type='secondary'>{ProgressModalDescription}</Typography.Text>
+                            </Space>
+                        </Col>
+                        <Col span={24}>
+                            <Progress
+                                format={(percent) => <Typography.Text strong>{Math.floor(percent)}%</Typography.Text>}
+                                percent={progress}
+                                status='active'
+                                strokeColor={colors.brandGradient[5]}
+                            />
+                        </Col>
+                    </Row>
+                </StyledFocusContainer>
+            </Modal>
+            <Modal
+                title={PartlyDataLoadedModalTitle}
+                open={activeModal === 'partlyLoaded'}
+                onCancel={closeModal}
+                footer={
+                    <Button
+                        type='primary'
+                        onClick={handleDownloadPartyLoadedData}
+                    >
+                        {GetFailedDataMessage}
+                    </Button>
+                }
+            >
+                <Alert
+                    type='warning'
+                    showIcon
+                    message={
+                        PartlyDataLoadedModalAlertMessage
+                            .replace('{success}', successRowsRef.current)
+                            .replace('{total}', totalRowsRef.current)
+                            .replace('{genitive}', ImportGenitiveMessage.toLowerCase())
+                    }
+                    description={PartlyDataLoadedModalAlertDescription}
+                />
+            </Modal>
+            <Modal
+                title={ErrorModalTitle}
+                open={activeModal === 'error'}
```

Comment:
Maybe move all activemodals to enum for ease of reading

Reply:
Yes, that's right

<img width="1050" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/e21cc7fb-072a-480a-88b8-f8ca8ddf0f91">
---
<img width="920" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/5a453e19-4d60-4d6c-8ac7-4ed35a8ec633">